content/en/service_catalog/adding_metadata.md

```diff
@@ -468,4 +476,6 @@ extensions:
 [14]: https://registry.terraform.io/providers/DataDog/datadog/latest/docs/resources/service_definition_yaml
 [15]: https://github.com/DataDog/schema/blob/main/service-catalog/v2.2/schema.json
 [16]: https://github.com/DataDog/schema/tree/main/service-catalog/v3
-
+[17]: http://json-schema.org/
+[18]: https://www.schemastore.org/json/
+[19]: https://raw.githubusercontent.com/DataDog/schema/main/service-catalog/version.schema.json
```

Comment:
This link seems to be broken? I'm getting a 404.

Reply:
Is this the right link: https://github.com/DataDog/schema/blob/main/service-catalog/service.schema.json ??